src/modules/pageTable.js

```diff
@@ -27,10 +34,26 @@ class PageTableRow extends React.PureComponent {
                         accessor = column.accessor(row);
                     }
 
+                    const style = {};
+                    const size = (sizes[rowIndex] || [])[index];
```

Comment:
The ` || []` fallback and then indexing it seems a bit odd to me as clearly that would result in `undefined`.  That's probably perfectly OK ... one of those instances where JavaScript allows you to do things that would blow up loudly in C# so it messes with my head a bit. =)  Probably there nothing to do here, but I'm just flagging it for discussion, in case there is a "nicer" way to default this if `sizes` 2-d array is undefined or empty or what-not.

Reply:
Fixed